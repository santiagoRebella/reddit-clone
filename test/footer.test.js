const React = require('react');
const Footer = require('components/footer');
const renderer = require('react-test-renderer');

test('Footer is rendered correctly', () => {
  const component = renderer.create(
    <Footer />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
