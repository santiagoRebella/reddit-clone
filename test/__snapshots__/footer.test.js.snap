// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer is rendered correctly 1`] = `
<footer
  className="row"
  style={
    Object {
      "bottom": 10,
      "position": "fixed",
      "right": 10,
    }
  }
>
  <div
    className="col-sm-12 float-right text-sm-right"
  >
    <button
      className="btn btn-link"
      onClick={[Function]}
      type="button"
    >
      go up
    </button>
  </div>
</footer>
`;
